homeassistant/components/auth/__init__.py

```diff
@@ -651,10 +651,9 @@ def websocket_delete_all_refresh_tokens(
             continue
         try:
             hass.auth.async_remove_refresh_token(token)
-        except Exception as err:  # pylint: disable=broad-except
+        except Exception:  # pylint: disable=broad-except
             getLogger(__name__).exception(
-                "During refresh token removal, the following error occurred: %s",
-                err,
+                "During refresh token removal, the following error occurred"
```

Comment:
Please take a look at the test case for it. This change seems to cause a failure. It's likely not enough to check in `caplog.record_tuples`. You can find the `exc_info` in `caplog.records[1]` in this case.

_Or just leave it as is and add a noqa._